src/clj_rn/core.clj

```diff
@@ -178,10 +178,16 @@
        (future (shell/exec full-command))
        (shell/exec full-command)))))
 
+(defn get-run-options
+  [build-id config]
+  (or (get-in config [:platforms build-id :run-options]) (:run-options config)))
```

Comment:
maybe better to merge them?

Reply:
How about this?
```clojure
(defn- get-platform-config
  [config platform]
  (merge config (get-in config [:platforms platform])))

(defn- run-builds
  [build-ids config]
  (doseq [build-id build-ids]
    (let [platform-config (get-platform-config config build-id)
          command (str "run-" (name build-id) " " (:run-options platform-config))]
      (execute-react-native-cli command))))

```

Reply:
i'm not sure if `merge` works for strings, and see Julien comment below

Reply:
It doesn't, but I don't merge strings here. Platform config is a map.

Reply:
ok cool then